Makefile

```diff
@@ -41,11 +41,21 @@ test-moParser:
 generate-reference-output:
 	(cd test/FromModelica && \
 	for ff in `find . -name '*.mo'`; do \
-		node ../../app.js -l warn -f $${ff} -o json; \
-		node ../../app.js -l warn -f $${ff} -o json-simplified; \
-		node ../../app.js -l warn -f $${ff} -o html; \
+		node ../../app.js -l warn -f $${ff} -o json -m cdl; \
+		node ../../app.js -l warn -f $${ff} -o json-simplified -m cdl; \
+		node ../../app.js -l warn -f $${ff} -o html -m cdl; \
 		done)
-	rm -f test/FromModelica/modelica-json.log
+	cp -r test/FromModelica/json test/FromModelica/cdl && rm -R test/FromModelica/json
```

Comment:
Introduce a flag `-d|--directory` to specify the output directory, with the default being the current directory.